src/http/cf.rs

```diff
@@ -38,6 +39,17 @@ pub fn featured_cf_v4_client(
     )
 }
 
+pub fn cf_v4_api_client_async(
+    user: &GlobalUser,
+    config: HttpApiClientConfig,
+) -> Result<async_api::Client, failure::Error> {
+    async_api::Client::new(
+        Credentials::from(user.to_owned()),
+        config,
+        Environment::Production,
```

Comment:
What is defined as production environment?

Reply:
this is part of cloudflare-rs; it's configurable so you can change the api hostname from the production api (api.cloudflare.com) to something usable for testing, be it a staging env or a custom instance of something running locally.